core/src/main/java/com/alibaba/nacos/core/distributed/distro/task/execute/AbstractDistroExecuteTask.java

```diff
@@ -130,7 +130,7 @@ public void onFailed(Throwable throwable) {
             if (null == throwable) {
                 Loggers.DISTRO.info("[DISTRO-END] {} result: false", getDistroKey().toString());
             } else {
-                Loggers.DISTRO.warn("[DISTRO] Sync data change failed.", throwable);
+                Loggers.DISTRO.warn("[DISTRO] Sync data change failed. key: {}", getDistroKey(), throwable);
```

Comment:
```suggestion
                Loggers.DISTRO.warn("[DISTRO] Sync data change failed. key: {}", getDistroKey().toString(), throwable);
```

I think it is better